laterpay/asset_sources/js/laterpay-backend-contributions.js

```diff
@@ -223,7 +223,10 @@
                                 $($o.revenueModelMultipleItems).each(function (idx) {
                                     var inputId = idx + 1;
                                     var $inputElement = $('#lp_multiple_contribution_input_' + inputId);
-                                    var price = $inputElement.val().length ? $inputElement.val() : 0.00;
+
+                                    var price = parseInt( $inputElement.val(), 10 );
```

Comment:
I would personally have these magic numbers in variables but I know this is an old codebase so will leave that up to you.